index.html

```diff
@@ -4,10 +4,123 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Portfolio</title>
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Poppins" />
+  <!-- including css -->
+  <link rel="stylesheet" href="./CSS/index.css">
+  <link rel="stylesheet" href="./CSS/navbar.css">
+  <link rel="stylesheet" href="./CSS/images.css">
+  <link rel="stylesheet" href="./CSS/aboutme.css">
+  <link rel="stylesheet" href="./CSS/connections.css">
 </head>
 
 <body>
+ 
+    <nav class="navbar">
+      <div class="logo">MyLogo</div>
+
+      <div class="menu-burger-icon">
+        <div class="bar"></div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+      </div>
+    </nav>
+
+    <section >
+      
+      <div class="img ecllipse1-img">
+        <img src="./assets/images/Ellipse-3.png" alt="Elipse">
+      </div>
+
+      <div class="img block1-img">
+        <img src="./assets/images/Rectangle-4.png" alt="Elipse">
+      </div>
+
+      <div class="img block2-img">
+        <img src="./assets/images/Rectangle-6.png" alt="Elipse">
+      </div>
+
+      <div class="img ecllipse2-img">
+        <img src="./assets/images/Ellipse-11.png" alt="Elipse">
+      </div>
+
+      <div class="img block3-img">
+        <img src="./assets/images/Rectangle-7.png" alt="Elipse">
+      </div>
+
+      <div class="img circle1-img">
+        <img src="./assets/images/Ellipse-5.png" alt="Elipse">
+      </div>
+
+      <div class="img circle2-img">
+        <img src="./assets/images/Ellipse-8.png" alt="Elipse">
+      </div>
+
+      <div class="img circle3-img">
+        <img src="./assets/images/Ellipse-14.png" alt="Elipse">
+      </div>
+      <div class="img circle4-img">
+        <img src="./assets/images/Ellipse-12.png" alt="Elipse">
+      </div>
+
+      <div class="img ecllipse3-img">
+        <img src="./assets/images/Ellipse-13.png" alt="Elipse">
+      </div>
+      
+      <div class="img block4-img">
+        <img src="./assets/images/Rectangle-5.png" alt="Elipse">
+      </div>
+
+      <div class="img circle5-img">
+        <img src="./assets/images/Ellipse-4.png" alt="Elipse">
+      </div>
+      
+      <div class="img ecllipse4-img">
+        <img src="./assets/images/Ellipse-15.png" alt="Elipse">
+      </div>
```

Comment:
- [ ] Kindly remove the background images from HTML code, a background image should be added in CSS as the `background-image` of the section. hint: you can download the whole background image from Figma instead of downloading the pieces one by one (check the left sidebar) 👍🏻